pkg/miniogw/gateway-storj.go

```diff
@@ -48,7 +217,8 @@ func (s *Storj) Production() bool {
 }
 
 type storjObjects struct {
-	cmd.GatewayUnsupported
+	minio.GatewayUnsupported
+	TempDir string // Temporary storage location for file transfers.
```

Comment:
also maybe have a reference to `*Storj` here, like

```
type storjObjects struct {
   minio.GatewayUnsupported
   TempDir string
   s *Storj
}
```

Reply:
Modified as suggested to eliminate global references